### ISSUE TITLE
metadata in ledger.transactions[].transaction is null 

### ISSUE THREAD

**Issue by manthanhd** [XRPLF/xrpl4j#286]
Seeing a strange serialization behaviour on the ledger rippled method via XRPL4J. I'm doing the following:

```java
XrplClientProvider provider = new XrplClientProvider();
LedgerResult ledger = provider.getClient(BlockchainNetwork.TESTNET)
    .ledger(LedgerRequestParams.builder()
        .ledgerIndex(LedgerIndex.of(UnsignedInteger.valueOf(29487937L)))
        .transactions(true)
        .build());
```

From what I see, although I cannot set the `LedgerRequestParams.builder().expand(true)` property directly, it is derived from the `.transactions(true)` property so it is receiving expanded set of transactions. When I view the data in debugger I can see that the transaction data is indeed "full" containing the `account`, `destination` etc properties.

However, the `metadata` is null and missing. I can only get this when I explictly call the `transaction` rippled method on each and every single transaction hash contained in the requested ledger. This seems like a strange behaviour because when I look on the ledger directly, the `metadata` is in the response of the `ledger` method but it is inside the transaction object rather than outside it. So at a glance this seems like a simple serialisation issue that you might be able to help patch.

While the current code works, it seems overkill to call `transaction` method on every single transaction just to get the `metadata` especially when the whole transaction object is already being returned. So ideally for me I would just prefer the serialisation to be fixed so that on ledger I can get the `metadata` directly.

**Comment by manthanhd** [comment]
Actually this might be a simpler problem than I thought. The `metadata` field is called `metaData` in `ledger` response but is called `meta` in `transaction` response. Provided same class is used in both cases, it could just be an issue with field naming.

**Comment by sappenin** [comment]
This is fixed via #288.
